raspibolt/raspibolt_6B_lndhub.md

```diff
@@ -121,7 +121,9 @@ $ wget https://nodejs.org/dist/v10.15.1/node-v10.15.1-linux-armv7l.tar.xz
 ```
 Extract to `usr/local/lib`
 ```
+$ tar -xf node-v10.15.1-linux-armv7l.tar.xz
 $ sudo mkdir /usr/local/lib/nodejs/node-v10.15.1
+$ mv node-v10.15.1-linux-armv7l.tar.xz/* /usr/local/lib/nodejs/node-v10.15.1
 $ sudo tar -xf node-v10.15.1-linux-armv7l.tar.xz -C /usr/local/lib/nodejs/node-v10.15.1
```

Comment:
Should remove this line, as a secondary extract of the tar file to **/usr/local/lib/nodejs/node-v10.15.1** is no longer needed